java_8/src/test/java/android/BrowserStackSample.java

```diff
@@ -27,57 +27,50 @@ public static void main(String[] args)
     browserstackOptions.put("appiumVersion", "1.22.0");
     browserstackOptions.put("projectName", "First Java Project");
     browserstackOptions.put("buildName", "browserstack-build-1");
-    browserstackOptions.put("sessionName", "first_test");
+    browserstackOptions.put("sessionName", "BStack first_test");
 
     // Passing browserstack capabilities inside bstack:options
-    caps.setCapability("bstack:options", browserstackOptions);
+    options.setCapability("bstack:options", browserstackOptions);
 
     // Set URL of the application under test
-    caps.setCapability("app", "bs://<app-id>");
+    options.setCapability("app", "bs://<app-id>");
 
     // Specify deviceName and platformName for testing
-    caps.setCapability("deviceName", "Google Pixel 3");
-    caps.setCapability("platformName", "android");
-    caps.setCapability("platformVersion", "9.0");
+    options.setCapability("deviceName", "Google Pixel 3");
+    options.setCapability("platformName", "android");
+    options.setCapability("platformVersion", "9.0");
```

Comment:
This is non-w3c compliant and it must go inside bStackOptions ?
https://www.w3.org/TR/webdriver1/#capabilities

Reply:
For Appium these caps are general capabilities and not specific to Browserstack, please ref [here](https://appium.io/docs/en/writing-running-appium/caps/#general-capabilities)